docs/components/link.md

```diff
@@ -16,6 +16,24 @@ import { Link } from "@remix-run/react";
 
 ## Props
 
+### `to: string`
+The most basic usage takes an href string:
+```tsx
+<Link to="/some/path" />
+```
+
+### `to: Path`
+You can also pass a `Path` value:
```

Comment:
If we're going to expose the underlying type here, let's also expose the `Partial` aspect?

```suggestion
### `to: Partial<Path>`
You can also pass a `Partial<Path>` value:
```